lib/before-watchPatterns.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function (hookArgs) {
+	if (hookArgs.liveSyncData && hookArgs.liveSyncData.bundle) {
+		const appDirectoryLocation = "app"; // Might need to get this from config file in the future
+		const appResourcesDirectoryLocation = "app/App_Resources"; // Might need to get this from config file in the future
+		return (args, originalMethod) => {
+			return originalMethod().then(originalPatterns => {
+				const appDirectoryLocationIndex = originalPatterns.indexOf(appDirectoryLocation);
+				if (appDirectoryLocationIndex !== -1) {
+					originalPatterns.splice(appDirectoryLocationIndex, 1);
+				}
+
+				if (originalPatterns.indexOf(appResourcesDirectoryLocation) === -1) {
+					originalPatterns.push(appResourcesDirectoryLocation);
+				}
```

Comment:
I think it would be better to keep the appResourcesDirectory in the [original watch patterns](https://github.com/NativeScript/nativescript-cli/pull/3350/files#diff-ce2a772bf174b37fc93372c01de3302cR295), instead of adding it manually in the bundler.